statsmodels/tsa/statespace/tests/test_sarimax.py

```diff
@@ -0,0 +1,1149 @@
+"""
+Tests for SARIMAX models
+
+Author: Chad Fulton
+License: Simplified-BSD
+"""
+from __future__ import division, absolute_import, print_function
+
+import numpy as np
+import pandas as pd
+import os
+
+import warnings
+from statsmodels.tsa.statespace import sarimax, tools
+from .results import results_sarimax
+from statsmodels.tools import add_constant
+from numpy.testing import assert_almost_equal, assert_raises, assert_allclose
+from nose.exc import SkipTest
+
+current_path = os.path.dirname(os.path.abspath(__file__))
+
+coverage_path = 'results' + os.sep + 'results_sarimax_coverage.csv'
+coverage_results = pd.read_csv(current_path + os.sep + coverage_path)
+
+
+class SARIMAXStataTests(sarimax.SARIMAX):
```

Comment:
better to attach the estimated model so we can separate better what is the unit test and what is the model.